ng-dev/format/format.ts

```diff
@@ -19,7 +19,7 @@ export async function formatFiles(files: string[]) {
 
```

Comment:
I can't comment on the `formatFiles` function signature, but we should add an explicit return type so we never can accidentally return without any value/or a different value. + a `@returns` JSDoc would be nice to have.